SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -204,6 +204,15 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Local Connection")
 	FString DefaultReceptionistHost;
 
+private:
+	/** Will stop a non editor client auto connecting via command line args to a cloud deployment */
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection")
+	bool bPreventClientCloudDeploymentAutoConnect;
+
+public:
+
+	bool GetPreventClientCloudDeploymentAutoConnect(bool bIsClient) const { return bIsClient && bPreventClientCloudDeploymentAutoConnect; };
```

Comment:
I'm not sure this function should be in settings - I'd keep it in SpatialNetDriver

Reply:
Sure, it was just so that if anywhere else where to need the setting in the future it would ensure that people have to realize that they need to consider whether we are running as client or server. I could imagine someone simply using the value of bPreventClientCloudDeploymentAutoConnect and messing up. But I also don't particularly like it here either. Having the function on NetDriver is a bit of a compromise but it doesn't protect against use of  bPreventClientCloudDeploymentAutoConnect outside of NetDriver. Anyway, I have presented my case. I will let yo have the final say.